src/Codeception/Event/SuiteEvent.php

```diff
@@ -22,7 +22,7 @@ class SuiteEvent extends Event
     protected $result;
 
     /**
-     * @var array
+     * @var array|null
      */
     protected $settings = [];
 
```

Comment:
Yeah... Someone is also waiting for `getSuite` to return... a DateTime? See [this](https://github.com/Codeception/Codeception/blob/db9c5b4aa30f92f8e03dc24a5a22002cb2d1e86c/tests/unit/Codeception/Subscriber/ModuleTest.php#L44). And failed build [here](https://github.com/Codeception/Codeception/pull/6137/checks?check_run_id=2086207496).

Reply:
It is a stupid test with fake mocks.
I disabled it in PHPUnit 10 branch, because Prophet is no longer a dependency of PHPUnit and it doesn't make sense to add it as dependency of Codeception for single test.

Reply:
https://github.com/Codeception/Codeception/blob/5.0-phpunit-10/tests/unit/Codeception/Subscriber/ModuleTest.php#L29

Reply:
Well, I added the `markTestIncomplete` line here for this to work.